src/main/java/com.flowkey.uikit/UserDefaults.kt

```diff
@@ -0,0 +1,39 @@
+package com.flowkey.uikit
+
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
+
+private const val FILE_NAME = "com.flowkey.uikit.UserDefaults"
+
+@Suppress("unused")
+class UserDefaults(private val context: Context) {
+    private val preferences = context.getSharedPreferences(FILE_NAME, MODE_PRIVATE)
+
+    fun has(key: String): Boolean {
+        return preferences.contains(key)
+    }
+
+    fun getInt(key: String, defValue: Int): Int {
+       return preferences.getInt(key, defValue)
+    }
+
+    fun getBoolean(key: String, defValue: Boolean): Boolean {
+        return preferences.getBoolean(key, defValue)
+    }
+
+    fun getString(key: String, defValue: String): String {
+        return preferences.getString(key, defValue) ?: defValue
```

Comment:
This written a bit weird to make the compiler happy. It requires us to provide a default value but infers the return type of `preferences.getString()` to be `String?` :trollface: